package.json

```diff
@@ -56,6 +56,7 @@
     "grunt-coffeelint": "0.0.16",
     "grunt-contrib-coffee": "^2.0.0",
     "grunt-shell": "^1.3.0",
-    "jasmine-focused": ">=1.0.7 <2.0"
+    "jasmine-focused": ">=1.0.7 <2.0",
+    "node-gyp": "^7.1.0"
```

Comment:
- IMO `node-gyp` in `devDependencies` should be set to an exact version, (no caret `^` semver range) so it's stable forever.

- Or it can be removed from `devDependencies`, and just allow `require.resolve('node-gyp')` to find the indirect dependency from `npm` --> `node-gyp`.